src/stateFromElement.js

```diff
@@ -260,6 +276,9 @@ class BlockGenerator {
     if (element.childNodes != null) {
       Array.from(element.childNodes).forEach(this.processNode, this);
     }
+    if (SELF_CLOSING_ELEMENTS.hasOwnProperty(tagName)) {
+      this.processText('~');
```

Comment:
Will move this to options then :), any suggestions about how to name this option tho?